src/ServiceManagement/Network/Network.Tests/Utilities.cs

```diff
@@ -0,0 +1,253 @@
+    using Microsoft.WindowsAzure.Storage.Blob;
+    using Microsoft.WindowsAzure.Storage.Auth;
+    using System.Collections.Generic;
+
+    public static class Utilities
+    {
+        public static void CreateStorageAccount(string location, StorageManagementClient storageClient,
+            string storageAccountName, out bool storageAccountCreated)
+        {
+            AzureOperationResponse storageCreate = storageClient.StorageAccounts.Create(
+                new StorageAccountCreateParameters
+                {
+                    Location = location,
+                    Label = storageAccountName,
+                    Name = storageAccountName,
+                    AccountType = "Standard_LRS"
+                });
+            Assert.True(storageCreate.StatusCode == HttpStatusCode.OK);
+            storageAccountCreated = true;
+        }
+
+        public static void CreateHostedService(string location, ComputeManagementClient computeClient, string serviceName,
+            out bool hostedServiceCreated)
+        {
+            AzureOperationResponse hostedServiceCreate = computeClient.HostedServices.Create(
+            new HostedServiceCreateParameters
+            {
+                Location = location,
+                Label = serviceName,
+                ServiceName = serviceName
+            });
+
+            Assert.True(hostedServiceCreate.StatusCode == HttpStatusCode.Created);
+
+            hostedServiceCreated = true;
+        }
+
+        public static string GetTestLocation(ManagementClient managementClient)
+        {
+            string location = managementClient.GetDefaultLocation("Storage", "Compute");
+            const string usWestLocStr = "West US";
+            if (managementClient.Locations.List().Any(
+                c => string.Equals(c.Name, usWestLocStr, StringComparison.OrdinalIgnoreCase)))
+            {
+                location = usWestLocStr;
+            }
+            return location;
+        }
+
+        public static void CreateAzureVirtualMachine(ComputeManagementClient computeClient, string serviceName, string deploymentName, string storageAccountName, string blobUrl)
+        {
+            VirtualMachineOSImageListResponse imagesList = computeClient.VirtualMachineOSImages.List();
+
+            VirtualMachineOSImageListResponse.VirtualMachineOSImage imageToGet =
+                imagesList.Images.FirstOrDefault(i => string.Equals(i.OperatingSystemType, "Windows", StringComparison.OrdinalIgnoreCase));
+
+            VirtualMachineOSImageGetResponse gottenImage = computeClient.VirtualMachineOSImages.Get(imageToGet.Name);
+
+            VirtualMachineCreateDeploymentParameters parameters = CreateVMParameters(gottenImage, deploymentName, storageAccountName, "SampleLabel", blobUrl);
+
+            parameters.Roles[0].ConfigurationSets.Add(new ConfigurationSet
+            {
+                AdminUserName = "testuser",
+                AdminPassword = "@zur3R0ck5",
```

Comment:
What is this user name and password? Are they used against any existing configuration?